### PR TITLE
Some editorial changes

### PR DESCRIPTION
**Pull request by chrisn**
I've just had a quick read through the PiP spec, looks great so far! I made a few editorial changes as I was reading. The main one is that some text and the examples used the term "Picture-in-Picture mode", but elsewhere this is simply "Picture-in-Picture", so I edited to use "Picture-in-Picture mode" throughout. I hope these changes are OK with you.

***
<a href="https://pr-preview.s3.amazonaws.com/chrisn/picture-in-picture/pull/118.html" title="Last updated on Feb 8, 2019, 6:08 PM UTC (81a101c)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/WICG/picture-in-picture/118/b926a27...chrisn:81a101c.html" title="Last updated on Feb 8, 2019, 6:08 PM UTC (81a101c)">Diff</a>

### CODE REVIEW

File: index.bs [hunk 1]
```diff
@@ -134,6 +140,7 @@ extensible.
 
 <script>
   let pipWindow;
+  const pipButton = document.getElementById('pipButton');
```
Comment (beaufortfrancois) [inline index.bs]: Shall we add `const video = document.getElementById('video');`as well then?

**Comment by chrisn** [comment]
Many thanks, @beaufortfrancois. You're right, will do.

File: index.bs (hunk 1 above)
Comment (chrisn) [reply index.bs]: Sorry, I missed that... I just pushed a fix, please take a look!

**Comment by beaufortfrancois** [comment]
Thank you again @chrisn!

**Pull request merged**